tools/packaging/static-build/cloud-hypervisor/build-static-clh.sh

```diff
@@ -18,7 +18,27 @@ cloud_hypervisor_version="${cloud_hypervisor_version:-}"
 [ -n "$cloud_hypervisor_version" ] || cloud_hypervisor_version=$(get_from_kata_deps "assets.hypervisor.cloud_hypervisor.version" "${kata_version}")
 [ -n "$cloud_hypervisor_version" ] || die "failed to get cloud_hypervisor version"
 
-info "Download cloud-hypervisor version: ${cloud_hypervisor_version}"
-cloud_hypervisor_binary="https://github.com/cloud-hypervisor/cloud-hypervisor/releases/download/${cloud_hypervisor_version}/cloud-hypervisor-static"
+pull_clh_released_binary() {
+    info "Download cloud-hypervisor version: ${cloud_hypervisor_version}"
+    cloud_hypervisor_binary="https://github.com/cloud-hypervisor/cloud-hypervisor/releases/download/${cloud_hypervisor_version}/cloud-hypervisor-static"
 
-curl --fail -L ${cloud_hypervisor_binary} -o cloud-hypervisor
+    curl --fail -L ${cloud_hypervisor_binary} -o cloud-hypervisor || return 1
+}
+
+build_clh_from_source() {
+    info "Build ${cloud_hypervisor_repo} version: ${cloud_hypervisor_version}"
+    repo_dir=$(basename "${cloud_hypervisor_repo}")
+    repo_dir="${repo_dir//.git}"
+    [ -d "${repo_dir}" ] || git clone "${cloud_hypervisor_repo}"
+    cd "${repo_dir}"
```

Comment:
I'd prefer to use `pushd` and `popd` before returning

Reply:
Good point. Just updated the PR. Thanks.